mms/arg_parser.py

```diff
@@ -117,8 +118,16 @@ def export_parser():
                                         'Alternatively, if the input defined in signature.json is image/jpeg '
                                         'this tool will include the MXNetVisionService in the archive.')
 
+        parser_export.add_argument('--model-type',
```

Comment:
Can we avoid adding this, and simplify the tool via other means?
I suggested considering using custom service class hierarchy, please consider this and maybe other ideas

Reply:
Sure. We can discuss this as a team and figure out a better solution. As of now, this is an optional parameter which defaults to using symbols. 

I couldn't think of a conclusive way to infer that the model is imperative from the model artifacts. It would be great if we could infer this from the artifacts as well.

Reply:
I agree with you on inferring the models types based on the base class used. I will make changes for that and default the behavior to include all the files in "model-path".